tests/__mocks__/src/utils/delayUtil.ts

```
@@ -0,0 +1,4 @@
+export default async function delayFrame() {
+  return Promise.resolve();
+  return Promise.resolve();
```

Biome (2.4.4)

[error] This code will never be reached ...

(lint/correctness/noUnreachable)